ACTIONS-FILTERS.md

```diff
@@ -200,7 +200,7 @@ add_filter( 'convertkit_for_woocommerce_order_maybe_subscribe_customer_subscript
 						convertkit_for_woocommerce_order_send_purchase_data
 						<code>includes/class-ckwc-order.php::499</code>
 					</h3><h4>Overview</h4>
-						<p>Define the data to send to the ConvertKit API to create a Purchase in ConvertKit https://developers.convertkit.com/#create-a-purchase</p><h4>Parameters</h4>
+						<p>Define the data to send to the ConvertKit API to create a Purchase in ConvertKit https://developers.kit.com/#create-a-purchase</p><h4>Parameters</h4>
```

Comment:
Are you planning on another PR to do the text/branding updates? This line still says ConvertKit API

Reply:
Each Plugin has a separate PR for branding (e.g. https://github.com/ConvertKit/convertkit-woocommerce/pull/195)